src/Core/Entities/Organization.cs

```diff
@@ -72,6 +72,11 @@ public class Organization : ITableObject<Guid>, ISubscriber, IStorable, IStorabl
     public int? MaxAutoscaleSeats { get; set; } = null;
     public DateTime? OwnersNotifiedOfAutoscaling { get; set; } = null;
     public OrganizationStatusType Status { get; set; }
+    public bool UsePasswordManager { get; set; } = true;
```

Comment:
Why not add a column for `UseSecretsManager`? Do we instead check if `SmSeats > 0`?

Reply:
`AccessSecretsManager` already exists as a column.

Reply:
Yes but that column is on the `OrganizationUser` table, shouldn't they be configured the same way? Is SecretsManager enabled for each user whereas PasswordManager is configured for each organization?

Reply:
here is already a column called UseSecretsManager in the Organization.cs class

Reply:
@r-tome I was getting mixed up between tables, just to restate this a bit clearer:
* `Organization.UseSecretsManager` indicates whether the org has SM enabled
* `OrganizationUser.AccessSecretsManager` indicates whether the user has access to SM in their organization

These both already exist. This PR is adding these columns for PM as well.